modules/sync/syncs/urlsynchronization.cpp

```diff
@@ -225,7 +225,7 @@ bool UrlSynchronization::isEachFileValid() {
 
     if (ossyncVersion == "1.0") {
         // We need to mutex-protect the access to the time conversion for now
-        std::lock_guard guard(_mutex);
+        std::lock_guard guard(UrlSynchronization::_mutex);
```

Comment:
```suggestion
        std::lock_guard guard(_mutex);
```

No need to specify the class name directly, since this function is already part of the class